en/newbee/mod.md

```diff
@@ -2,47 +2,47 @@
 
-在一些情况下，安装皮肤 Mod 后，可能需要启动一次游戏并进入存档，Mod 才会自动生成配置文件；如果你在启动器中启用了版本隔离，配置文件的路径可能有所不同。
+In some cases, after you install your mod, you may have to launch the game and load a save to allow the mod to generate the configuration file. If you enabled the version isolation in your launcher, the file path of the configuration file may change.
 :::
 
 ::: danger
-只需要使用一种皮肤 Mod 即可。请不要同时安装多个皮肤 Mod，否则，轻则无法正常加载材质，重则导致游戏崩溃。
+You should only use one skin loader mod. Do not install multiple skin mods, otherwise, the skin might not load normally, it could even crash your game.
 :::
 
 ::: warning
-除 SkinPort 外，皮肤 Mod 和 Yggdrasil 外置登录二选一即可。请不要同时使用这两者，否则可能无法正常加载材质。
+Except for SkinPort, you should only use a skin mod or Yggdrasil. Do not enable both at the same time, otherwise, the skin might not load normally.
 :::
 
 ## CustomSkinLoader
 
-CustomSkinLoader 是我们最推荐的皮肤 Mod。
+CustomSkinLoader is our most recommended skin loader mod.
 
-你可以在 [MCBBS](https://www.mcbbs.net/thread-269807-1-1.html) 或 [CurseForge](https://www.curseforge.com/minecraft/mc-mods/customskinloader) 获取关于 CustomSkinLoader 的更多信息。
+You can get more information about CustomSkinLoader at [MCBBS](https://www.mcbbs.net/thread-269807-1-1.html) or [CurseForge](https://www.curseforge.com/minecraft/mc-mods/customskinloader).
 
 ### 14.7 +
 
-LittleSkin 自 CustomSkinLoader 14.7 起被添加到了 CustomSkinLoader 的默认加载列表中，加载顺序仅次于正版皮肤，在大部分情况下，安装完成后你无需进行任何修改即可加载来自 LittleSkin 的材质。Cheers!
+LittleSkin has been added to CustomSkinLoader's default loading list from version 14.7. It's just after the Mojang official loader. Usually, you don't need to change anything to load the textures from LittleSkin. Cheers!
 
-如果因为存在与你同名的正版角色导致冲突而无法加载来自 LittleSkin 的材质，请参考下方的适用于 CustomSkinLoader 13.1 ~ 14.6a 的配置方法来修改 CustomSkinLoader 的配置文件。
+If you are unable to load the texture from LittleSkin due to a conflict with a paid version character with the same name as you, please refer to the configuration method applicable to CustomSkinLoader 13.1 ~ 14.6a below to modify the configuration file of CustomSkinLoader.
 
 ### 13.1 ~ 14.6a
 
 #### ExtraList
 
-CustomSkinLoader 14.4 起支持通过 ExtraList 的方式添加皮肤站。你可以在用户中心的「皮肤 Mod」页面下载到 LittleSkin 的 ExtraList 文件，将其放入 `.minecraft/CustomSkinLoader/ExtraList/` 目录下即可。
+CustomSkinLoader supports adding the skin library using ExtraList from version 14.4. You can find LittleSkin's ExtraList file at “Generate Config” and put it under the file path `.minecraft/CustomSkinLoader/ExtraList/`.
 
-在安装完成后的第一次启动游戏并成功载入 CustomSkinLoader 时 ExtraList 文件会被自动删除，这是正常现象。如果不出意外的话，此时 LittleSkin 已被添加至 CustomSkinLoader 加载列表列表顶部。
+After the first successful launch of the game after you install the mod, the ExtraList file of CustomSkinLoader will be automatically deleted, this is normal. If everything's okay, LittleSkin has been added to the top of the CustomSkinLoader loading list.
 
-#### 手动修改配置文件
+#### Modify manually the configuration file
 
```

Comment:
::: tips
If you are not located in Mainland China, we suggest you use the Global CDN provided by Cloudflare. Just replace `mcskin.littleservice.cn` to `littleskin.cn`, so that you can get better experience.  

If you are located in Mainland China, you should use the CDN provided by Tencent Cloud, which has quicker cache refresh.
:::